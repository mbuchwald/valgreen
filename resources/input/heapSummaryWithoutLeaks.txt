==28== 
==28== HEAP SUMMARY:
==28==     in use at exit: 0 bytes in 0 blocks
==28==   total heap usage: 20,093 allocs, 20,093 frees, 1,738,508 bytes allocated
==28== 
==28== All heap blocks were freed -- no leaks are possible
==28== 
==28== For counts of detected and suppressed errors, rerun with: -v
==28== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)