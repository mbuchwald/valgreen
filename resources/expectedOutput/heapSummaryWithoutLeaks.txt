    [90mÂ [39m
    [90mHEAPÂ SUMMARY:[39m
    [90mÂ  Â  Â in use at exit: 0 bytes in 0 blocks[39m
    [90mÂ  Â total heap usage: 20,093 allocs, 20,093 frees, 1,738,508 bytes allocated[39m
    [90mÂ [39m
    [1m[32mAll heap blocks were freed -- noÂ leaksÂ are possible[39m[0m
    [90mÂ [39m
    [90mFor counts of detected and suppressed errors, rerun with: -v[39m
    [1m[32mERRORÂ SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)[39m[0m
